.github/workflows/release.yaml

```diff
@@ -43,6 +43,18 @@ jobs:
         sed -i 's/0.0.0/${{env.RELEASE_VER}}/g' src/robusta/_version.py pyproject.toml helm/robusta/Chart.yaml helm/robusta/values.yaml
         sed -i 's/0.0.1/${{env.RELEASE_VER}}/g' helm/robusta/Chart.yaml
 
+    # Set up the buildx to run build for multiple platforms
```

Comment:
Do we really need these 2 github actions?
Because I saw that in kubewatch, it works without it.
Why do we need it here?

Reply:
never mind, I understand why we need it here...